packages/client/src/components/editor.tsx

```diff
@@ -10,11 +10,12 @@ import { useSelector } from "react-redux";
 import { selectDocument } from "ducks/appState/document";
 import styled from "@emotion/styled";
 import { selectUser } from "ducks/appState/user";
+import DrawingCanvas from "./drawing-canvas";
 
 const SIGNALLING_SERVICE =
-  process.env.NEXT_PUBLIC_SIGNAL_URL || "ws://localhost:6006/ws/signal";
+  process.env.NEXT_PUBLIC_SIGNAL_URL || "ws://localhost:3000/ws/signal";
 const PROVIDER_SERVICE =
-  process.env.NEXT_PUBLIC_PROVIDER_URL || "ws://localhost:6006/ws/provider";
+  process.env.NEXT_PUBLIC_PROVIDER_URL || "ws://localhost:3000/ws/provider";
```

Comment:
NextJS isn't setup to forward ports the way storybook was. Which is why we have to run it behind a nginx proxy in development (docker).

When I tried to do it, the API routes forward correctly, websockets were trickier. Here is some documentation if you want to try:

https://nextjs.org/docs/api-routes/api-middlewares